src/main/java/chiikawa/Chiikawa.java

```diff
@@ -0,0 +1,411 @@
+package chiikawa;
+
+import java.nio.file.Path;
+import java.time.format.DateTimeParseException;
+
+import chiikawa.exception.ChiikawaException;
+import chiikawa.exception.EmptyDescriptionException;
+import chiikawa.exception.IndexOutOfBoundException;
+import chiikawa.exception.ListEmptyException;
+import chiikawa.exception.NoDeadlineException;
+import chiikawa.exception.NoEventException;
+import chiikawa.exception.NoIndexException;
+import chiikawa.task.Deadline;
+import chiikawa.task.Event;
+import chiikawa.task.Task;
+import chiikawa.task.Todo;
+import javafx.animation.PauseTransition;
+import javafx.application.Platform;
+import javafx.util.Duration;
```

Comment:
Maybe you could further break up the imports with line breaks for each parent class, and also your own classes vs the java and third party classes

```suggestion
import chiikawa.exception.ChiikawaException;
import chiikawa.exception.EmptyDescriptionException;
import chiikawa.exception.IndexOutOfBoundException;
import chiikawa.exception.ListEmptyException;
import chiikawa.exception.NoDeadlineException;
import chiikawa.exception.NoEventException;
import chiikawa.exception.NoIndexException;

import chiikawa.task.Deadline;
import chiikawa.task.Event;
import chiikawa.task.Task;
import chiikawa.task.Todo;

import javafx.animation.PauseTransition;
import javafx.application.Platform;
import javafx.util.Duration;
```